README.md

```diff
@@ -1,30 +1,100 @@
-# PegNet
-This is now the main repository for mining for the PegNet
+<p align="center">
+  <img src="https://pegnet.org/assets/img/logo.png"/>
+</p>
 
-The TestNet has started writing records in mid June.
+----
 
-This is a work in progress.  The TestNet is certain to reset the actual rewards a few times as we develop more code and adjust the approach.  None the less, it is a great help to have multiple people do mining.  If you would like to help, you can run a miner.  To do so, do these 10 things:
+[![Build Status](https://travis-ci.com/pegnet/pegnet.svg?branch=develop)](https://travis-ci.com/pegnet/pegnet)
+[![Discord](https://img.shields.io/discord/550312670528798755.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/V6T7mCW)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/pegnet/pegnet/blob/master/LICENSE)
 
-*Assumes Linux or use of a virtual machine running Linux*
 
-1. Clone the OracleRecord repository https://github.com/pegnet/pegnet
 
-2. Copy the defaultconfig.ini file from the OracleRecord repository to ~/.pegnet directory
+## A Network of Pegged Tokens
 
-3. Sign up for an API Key from https://currencylayer.com.
+This is the main repository for the PegNet application.
 
-4. Replace the Oracle.APILayerKey key with your API Key
+Pegged tokens reflect real market assets such as currencies, precious metals, cryptocurrencies, commodities, etc. The conversion rates on PegNet are determined by a decentralized set of miners who submit values based on current market data. These values are then graded and recorded in the Factom blockchain. 
```

Comment:
Minor rephrase: "These values are recorded in the Factom blockchain and then graded." the other way around could be misleading?

Reply:
True.

Was tempted to go lower level with a description but not sure how general it should be.

Reply:
A mention of hashing difficulty in grading is probably noteworthy.

Reply:
i think the grading process deserves its own separate explainer such as the eventual FATIP protocol description